src/main/java/org/broadinstitute/hellbender/engine/AssemblyRegionWalker.java

```diff
@@ -43,7 +39,7 @@
  * Internally, the reads are loaded in chunks called read shards, which are then subdivided into active/inactive regions
  * for processing by the tool implementation. One read shard is created per contig.
  */
-public abstract class AssemblyRegionWalker extends GATKTool {
+public abstract class AssemblyRegionWalker extends Walker {
```

Comment:
Update docs for `AssemblyRegionWalker.traverse()` to read "You should only override traverse() if you are writing a new walker base class in the engine package that extends this class. It is not meant to be overridden by tools outside of the engine package." instead of the current "Subclasses can override to provide own behavior"

Reply:
Fixed.